main/src/main/java/org/mobilitydata/gtfsvalidator/validator/DuplicateRouteNameValidator.java

```diff
@@ -98,6 +98,6 @@ public void validate(NoticeContainer noticeContainer) {
    */
   private boolean areRoutesFromSameAgency(
       final String routeAgencyId, final String otherRouteAgencyId) {
-    return routeAgencyId.equalsIgnoreCase(otherRouteAgencyId);
```

Comment:
I would add an explicit note in the Javadocs of this method that IDs are considered case-sensitive to call attention to it. I'm not sure there will be universal agreement on this.

Reply:
Updated :)